apps/web/src/components/ChainDropdown/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useErrors } from 'apps/web/contexts/Errors';
+import Dropdown from 'apps/web/src/components/Dropdown';
+import DropdownItem from 'apps/web/src/components/DropdownItem';
+import DropdownMenu from 'apps/web/src/components/DropdownMenu';
+import DropdownToggle from 'apps/web/src/components/DropdownToggle';
+import { useCallback } from 'react';
+import { Chain } from 'viem';
+import { useAccount, useSwitchChain } from 'wagmi';
+
+export function DropdownChainSwitcher({
+  chain,
+  currentChain,
+}: {
+  chain: Chain;
+  currentChain: Chain;
+}) {
+  const { switchChainAsync } = useSwitchChain();
+  const { logError } = useErrors();
+
+  const handleSwitchChain = useCallback(() => {
+    if (chain !== currentChain) {
+      switchChainAsync({ chainId: chain.id })
+        .then(() => console.log('chainSwitched'))
+        .catch((error) => logError(error, 'Failed to switch chain'));
```

Comment:
I think this could "fail" if the user's wallet doesn't support the network, and we prob don't want to log an error in that case. We also prob don't need the log statement above.

Reply:
we need to log the error for some wallets. removed other console.log